docs/node-operators/snapshots.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 6
+title: Using snapshots
+hide_title: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This section will take you through using the state snapshots, these allow a node to be start without having to replay the whole chain in order to catchup with the current block height of the network.
+
+A node can be started by using a local snapshot (state of the chain built by the current node) this can be useful when the node have been running with the network, have built their own state locally, but had to shutdown for maintenance or upgrade for example.
+
+The snapshots can also be retrieved from the network without previous local state, this would be useful when a node is joining the network after the bootstrap of the network.
+
+All not of the network will be taking snapshot of the state at the same block height, this is configured globally using the following network parameter:
+```Json
+{
+  "snapshot.interval.length": "10000",
+}
+```
+As of now the snapshots are configured to be taken every 10000 blocks on mainnet.
+
+## List available snapshots
+The vega toolchain offer a subcommand to list all snapshots available locally. Once your node have been running for a while, you should be able to see them using the following command:
+```
+vega snapshots --home=/path/to/vega/home
+Snapshots available: 2
+	Height 901, version: 4, size 92, hash: 562414bb5be3ccc8403fbd030d06eebc799bfef5ca25b02ad360fec349cb4bc8
+	Height 601, version: 3, size 92, hash: 72a2edd960cf3340ae94bf092991f923850738144789959124590675798fefd9
+```
+Here we can see that our node took 2 snapshots, at the block 901 and 601.
+
+You can set how many blocks you want the note to retain through the vega configuration:
+```Toml
+[Snapshot]
+  ...
+  KeepRecent = 10
+  ...
+```
+With this setting a node will always keep the last 10 snapshots in its database.
+
+## Using a local snapshot
+Using the `vega snapshots` command line we can get a list of all snapshots available locally.
+
+You can then start your node using a snapshot either directly from the command line or the configuration file by specifying the block height.
+
+From the command line:
+```
+vega node --home=/path/to/vega/home --snapshot.load-from-block-height=901
+```
+
+From the configuration file in the Snapshot section:
+```
+[Snapshot]
+  ...
+  StartHeight = 901
+  ...
+```
+
+## Snapshots from the network
+
+:::note
+When loading snapshots from the network, the steps down previously to load locally are not necessary. You will need to get the snapshots information from another
+node runner in the network (e.g: which block height a snapshot was taken).
+:::
+
+Tendermint offer the possibility to gossip snapshots taken by other nodes. This can be enabled via the tendermint configuration. You will also need the hash of the block at the height you want to load the snapshot, but also a list of trusted tendermint RPC server (the default port on the node should be 26657).
```

Comment:
```suggestion
Tendermint offers the possibility to gossip snapshots taken by other nodes. This can be enabled via the tendermint configuration. You will also need the hash of the block at the height you want to load the snapshot, but also a list of trusted Tendermint RPC servers (the default port on the node should be 26657).
```